src/test/java/com/google/cloud/tools/managedcloudsdk/ManagedCloudSdkIntegrationTest.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.managedcloudsdk;
+
+import com.google.cloud.tools.managedcloudsdk.command.CommandExecutionException;
+import com.google.cloud.tools.managedcloudsdk.command.CommandExitException;
+import com.google.cloud.tools.managedcloudsdk.command.CommandRunner;
+import com.google.cloud.tools.managedcloudsdk.components.SdkComponent;
+import com.google.cloud.tools.managedcloudsdk.install.SdkInstallerException;
+import com.google.cloud.tools.managedcloudsdk.install.UnknownArchiveTypeException;
+import java.io.IOException;
+import java.util.Arrays;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.rules.TestWatcher;
+import org.junit.runner.Description;
+
+/** Integration tests for {@link ManagedCloudSdk} and all supporting classes. */
+public class ManagedCloudSdkIntegrationTest {
```

Comment:
Is this the standard place to put integration tests? This test seems to take a very long time to complete.

Reply:
Yeah, it is a long test, and I don't know if it's an "integration test" exactly, but we do want it to run all time, maybe I'll rename it and leave it in.